### DISCUSSION TITLE
Why saving the file is very slow?

### DISCUSSION THREAD

**Discussion by azzamsa** [ananthakumaran/paisa#271] [Q&A] [answered]
Hi, there.

I have only 1k lines of file, and pressing the' save' button takes so long.
Is this expected? Am I missing something?

Thank you for Paisa 🥞

**Comment by obsilva** [comment, marked as answer]
In addition to writing the changes to the file, when saving the file Paisa also creates a backup copy and performs a `Journal Sync` so that your changes are reflected immediately. In my experience, it's the latter that takes a few seconds.

When I want to do changes without this annoyance I just don't save the file until I'm done or I edit it outside Paisa and then manually `Journal Sync` once.

**Reply by azzamsa** [reply]
Ah, thank you. I thought it was because of my big file. But the demo file is bigger.

> When I want to do changes without this annoyance I just don't save the file until I'm done or I edit it outside Paisa and then manually Journal Sync once.

Yeah, Now I did everything in hledger and put in Paisa when I am done. Just to visualize things.

Btw, thanks a lot for the info! 🥞
